community/examples/AMD/hpc-amd-slurm.yaml

```diff
@@ -110,6 +106,12 @@ deployment_groups:
             concretizer:
               unify: when_possible
       commands: |
+        # gcc 12.1.0 is known to have runtime failures with OpenFOAM 8
+        # gcc 10.3.0 is the earliest copy of gcc with Zen 3 support
+        spack install gcc@10.3.0 %gcc@4.8.5 target=x86_64
+        spack load gcc@10.3.0 %gcc@4.8.5 target=x86_64
+        spack compiler find --scope site
+
```

Comment:
You should also add a `spack unload gcc@10.3.0` here. Otherwise it can make some of your next commands more confusing.